examples/ssh/linux_iplatency.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Linux Monitor IP Latency
+ * Description: This script is designed to ping an IP address and retrieve the average latency and packet loss percentage.
+ *   
+ * Communication protocol is SSH
+ * 
+ * Tested on Linux Distributions:
+ *      - Ubuntu 20.04 LTS
+ * Shell Version:
+ *      - Bash 5.1.16
+ * 
+ * Creates a Custom Driver table with a list of IP addresses, their average latency, and their packet loss.
+ * 
+ **/
+
+var pktno = "2"; // Number of packets to send during the ping command.
+var ipAddresses = ["8.8.8.8", "1.1.1.1", "192.168.0.1", "192.168.0.1", "172.17.0.1"]; // List of IP addresses to ping and retrieve status for.
```

Comment:
Please leave just two entries.

Also, you have duplicated IP address and this crashes the current version execution